requirements.txt

```diff
@@ -1,6 +1,5 @@
-pandas
-xarray
-segyio
-pytest
-matplotlib
-pyvista
+pandas~=1.1.1
+xarray~=0.16.0
+numpy~=1.19.1
+pytest~=6.0.1
+setuptools~=46.4.0
```

Comment:
Funny each requirement has one version that does not work. Why is that?

Reply:
I think those were the requirements in June. I did not know that meant "except", I will remove the versions thanks for the hint